test/support/utils.rb

```diff
@@ -12,13 +12,52 @@ def type(command)
     end
 
     def create_message fail_msg, test_info
-      "#{fail_msg} on #{test_info.mode} mode\n[DEBUGGER SESSION LOG]\n> #{test_info.backlog.join('> ')}#{debuggee_backlog test_info}"
-    end
+      debugger_msg = <<~DEBUGGER_MSG.chomp
+        ======================
+        ▼  Debugger Session  ▼
+        ======================
+
+        > #{test_info.backlog.join('> ')}
+
+        ======================
+        ▲  Debugger Session  ▲
+        ======================
+      DEBUGGER_MSG
+
+      debuggee_msg =
+        if test_info.mode != 'LOCAL' && debuggee_backlog = collect_debuggee_backlog(test_info)
```

Comment:
Do we need `debuggee_backlog = collect_debuggee_backlog(test_info)` as a conditional expression?

Reply:
we don't. I've fixed it, thanks 👍